do.py

```diff
@@ -859,27 +859,27 @@ def help():
   build-matlab            - Build Matlab module
   build-matlab-sms        - Build SMS-EMOA example in Matlab
   build-octave            - Build Matlab module in Octave
-  build-python            - Build Python modules
-  build-python2           - Build Python 2 modules
-  build-python3           - Build Python 3 modules
-  install-postprocessing  - Install postprocessing (user-locally)
+  build-python*           - Build Python modules
```

Comment:
Point taken. I'm slightly uncomfortable with the fact that the explanations are of different length and some even span multiple lines, so placing the mark at the end of the explanation makes it even more inconvenient to distinguish which commands are affected by this feature at a glance, but for lack of good alternatives, I'll roll with it. Still, I suggest using `¹` instead of `^*`, just to make it look more like the familiar footnote, unless `^*` has a meaning I am unaware of.